### PR TITLE
fix(deps): upgrade to XLTS Angular 1.8.5

### PR DESCRIPTION
**Pull request by adinaiordache2017**
Upgrade to XLTS Angular 1.8.5

### CODE REVIEW

**Comment by tradeshiftcicomponents** [comment]
*Semantic release: ([Commit conventions](https://github.com/conventional-changelog-archived-repos/conventional-changelog-angular/blob/master/convention.md))*

 No new version will be released.
                    If a release is expected, please fix commit messages to align with appropriate format :arrow_right: [Commit conventions](https://github.com/conventional-changelog-archived-repos/conventional-changelog-angular/blob/master/convention.md)

**Comment by tradeshiftcicomponents** [comment]
*Semantic release: ([Commit conventions](https://github.com/conventional-changelog-archived-repos/conventional-changelog-angular/blob/master/convention.md))*

 No new version will be released.
                    If a release is expected, please fix commit messages to align with appropriate format :arrow_right: [Commit conventions](https://github.com/conventional-changelog-archived-repos/conventional-changelog-angular/blob/master/convention.md)

**Comment by tradeshiftcicomponents** [comment]
*Semantic release: ([Commit conventions](https://github.com/conventional-changelog-archived-repos/conventional-changelog-angular/blob/master/convention.md))*

 No new version will be released.
                    If a release is expected, please fix commit messages to align with appropriate format :arrow_right: [Commit conventions](https://github.com/conventional-changelog-archived-repos/conventional-changelog-angular/blob/master/convention.md)

**Comment by tradeshiftcicomponents** [comment]
*Semantic release: ([Commit conventions](https://github.com/conventional-changelog-archived-repos/conventional-changelog-angular/blob/master/convention.md))*

 No new version will be released.
                    If a release is expected, please fix commit messages to align with appropriate format :arrow_right: [Commit conventions](https://github.com/conventional-changelog-archived-repos/conventional-changelog-angular/blob/master/convention.md)

**Comment by tradeshiftcicomponents** [comment]
*Semantic release: ([Commit conventions](https://github.com/conventional-changelog-archived-repos/conventional-changelog-angular/blob/master/convention.md))*

 No new version will be released.
                    If a release is expected, please fix commit messages to align with appropriate format :arrow_right: [Commit conventions](https://github.com/conventional-changelog-archived-repos/conventional-changelog-angular/blob/master/convention.md)

**Comment by tradeshiftcicomponents** [comment]
*Semantic release: ([Commit conventions](https://github.com/conventional-changelog-archived-repos/conventional-changelog-angular/blob/master/convention.md))*

 No new version will be released.
                    If a release is expected, please fix commit messages to align with appropriate format :arrow_right: [Commit conventions](https://github.com/conventional-changelog-archived-repos/conventional-changelog-angular/blob/master/convention.md)

File: package.json [hunk 1]
```diff
@@ -13,7 +13,8 @@
 		"lint": "tradeshift-scripts lint",
 		"precommit": "tradeshift-scripts precommit",
 		"test": "tradeshift-scripts test",
-		"validate": "tradeshift-scripts validate"
+		"validate": "tradeshift-scripts validate",
+		"preinstall": "npx npm-force-resolutions"
```
Comment (tradeshift-larsbojensen) [inline package.json:17]: If you upgrade to node 16 (and npm 8), you can skip npx npm-force-resolutions and replace `resolutions` with `overrides`.

Comment (adinaiordache2017) [reply package.json]: Done, please check again.

**Comment by tradeshiftcicomponents** [comment]
*Semantic release: ([Commit conventions](https://github.com/conventional-changelog-archived-repos/conventional-changelog-angular/blob/master/convention.md))*

 No new version will be released.
                    If a release is expected, please fix commit messages to align with appropriate format :arrow_right: [Commit conventions](https://github.com/conventional-changelog-archived-repos/conventional-changelog-angular/blob/master/convention.md)

**Comment by tradeshiftcicomponents** [comment]
*Semantic release: ([Commit conventions](https://github.com/conventional-changelog-archived-repos/conventional-changelog-angular/blob/master/convention.md))*

 No new version will be released.
                    If a release is expected, please fix commit messages to align with appropriate format :arrow_right: [Commit conventions](https://github.com/conventional-changelog-archived-repos/conventional-changelog-angular/blob/master/convention.md)

**Comment by tradeshiftcicomponents** [comment]
*Semantic release: ([Commit conventions](https://github.com/conventional-changelog-archived-repos/conventional-changelog-angular/blob/master/convention.md))*

 No new version will be released.
                    If a release is expected, please fix commit messages to align with appropriate format :arrow_right: [Commit conventions](https://github.com/conventional-changelog-archived-repos/conventional-changelog-angular/blob/master/convention.md)

**Comment by tradeshiftcicomponents** [comment]
*Semantic release: ([Commit conventions](https://github.com/conventional-changelog-archived-repos/conventional-changelog-angular/blob/master/convention.md))*

 No new version will be released.
                    If a release is expected, please fix commit messages to align with appropriate format :arrow_right: [Commit conventions](https://github.com/conventional-changelog-archived-repos/conventional-changelog-angular/blob/master/convention.md)

**Comment by tradeshiftcicomponents** [comment]
*Semantic release: ([Commit conventions](https://github.com/conventional-changelog-archived-repos/conventional-changelog-angular/blob/master/convention.md))*

## 1.0.1 (https://github.com/tradeshift/ng-translate-t/compare/v1.0.0...v1.0.1) (2022-03-18)

### Bug Fixes

 - deps: upgrade to XLTS Angular 1.8.5 (45301ef (https://github.com/tradeshift/ng-translate-t/commit/45301ef0eecc8d2ff89c850d6f29a7b215f8ed81))

**Comment by adinaiordache2017** [comment]
npm publish

**Comment by tradeshiftcicomponents** [comment]
:package: **npm publish** :package:

Pre-release of `@tradeshift/ng-translate-t` successfully published :rocket:

```
npm install @tradeshift/ng-translate-t@0.0.0-07bfeee1aa704fa554a174e47c42e8ca0e64fe17
```

Happy testing :tophat:

**Comment by tradeshiftcicomponents** [comment]
*Semantic release: ([Commit conventions](https://github.com/conventional-changelog-archived-repos/conventional-changelog-angular/blob/master/convention.md))*

## 1.0.1 (https://github.com/tradeshift/ng-translate-t/compare/v1.0.0...v1.0.1) (2022-03-18)

### Bug Fixes

 - deps: upgrade to XLTS Angular 1.8.5 (07bfeee (https://github.com/tradeshift/ng-translate-t/commit/07bfeee1aa704fa554a174e47c42e8ca0e64fe17))

**Comment by tradeshiftcicomponents** [comment]
*Semantic release: ([Commit conventions](https://github.com/conventional-changelog-archived-repos/conventional-changelog-angular/blob/master/convention.md))*

## 1.0.1 (https://github.com/tradeshift/ng-translate-t/compare/v1.0.0...v1.0.1) (2022-03-18)

### Bug Fixes

 - deps: upgrade to XLTS Angular 1.8.5 (a66d1a7 (https://github.com/tradeshift/ng-translate-t/commit/a66d1a7e171d19b28b3bcdc8f38aca1c55d76fa5))

**Pull request merged**